packages/utils/src/cmds/transform_translate.js

```diff
@@ -0,0 +1,381 @@
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\\n"
+"Language-Team: LANGUAGE <LL@li.org>\\n"
+"Language: ${language}\\n"
+"MIME-Version: 1.0\\n"
+"Content-Type: text/plain; charset=UTF-8\\n"
+"Content-Transfer-Encoding: 8bit\\n"
+"X-Generator: Translate Toolkit 2.2.5\\n"
+"X-Pootle-Path: /${language}/${appName}/${language}.po\\n"
+"X-Pootle-Revision: 29879\n"
+`
+
+exports.command = 'transform_translate'
+exports.describe = 'Transform old translation file style to new style'
+exports.builder = {
+    appName: {
+        describe:
+            'The name of the app, must be lower case, use dashes instead of whitespaces and be prefixed with "-app". Normally this should the url part of the github repo (maintenance app => "maintenance-app")',
+        type: 'string',
+    },
+
+    primaryLanguage: {
+        describe: 'Primary language',
+        type: 'string',
+        default: 'en',
+    },
+
+    languages: {
+        describe:
+            'A comma separated list of languages to create files for, when omitted, all are used',
+        type: 'string',
+    },
+
+    deleteOld: {
+        describe:
+            'Deletes the old translation files, use `--no-delete-old` to keep the old files',
+        type: 'boolean',
+        default: 'true',
+    },
+
+    outDir: {
+        describe: 'Output directory for new translation files',
+        type: 'string',
+        default: TRANSLATION_OUT_DIR,
+    },
+
+    inDir: {
+        describe: 'Input directory where old translation files can be found',
+        type: 'string',
+        default: TRANSLATION_IN_DIR,
+    },
+
+    createOutDir: {
+        describe:
+            'Create the output dir if not existing, use `--no-create-out-dir` to exit with 1 when folder does not exist',
+        type: 'boolean',
+        default: 'true',
+    },
+
+    overrideExistingFiles: {
+        describe: 'Overriding the contents of existing translation files',
+        type: 'boolean',
+        default: 'false',
+    },
+
+    appendToExistingFiles: {
+        describe:
+            'Appends the new contents to existing translation files, can only be used when not using `--override-existing-files`',
+        type: 'boolean',
+        default: 'false',
+    },
+
+    logMissingKeys: {
+        describe:
+            'Log keys that are present in translation files which are not present in translation file of --primary-language',
+        type: 'boolean',
+        default: 'false',
+    },
+
+    deleteOldFiles: {
+        describe:
+            'Delete the old files that were transformed (will only delete files specified with the `--language` option when present)',
+        type: 'boolean',
+    },
+}
+exports.handler = argv => {
+    const inDirExists = fs.existsSync(argv.inDir)
+    const outDirExists = fs.existsSync(argv.outDir)
+    const languagesToTransform = argv.languages
+        ? argv.languages.split(/,\s*/)
+        : []
+
+    if (!argv.appName) {
```

Comment:
Instead of checking for required args manually, pass `required: true` to the option builder.